openshift/templates/dps-payment-service.bc.yaml

```diff
@@ -0,0 +1,71 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: dps-payment-service-build
+  annotations:
+    openshift.io/display-name: "DPS Payment Service Build"
+    description: The DPS Payment Service builder
+    tags: java
+    template.openshift.io/provider-display-name: BCGov
+    template.openshift.io/documentation-url: https://github.com/bcgov/jag-dps/
+
+parameters:
+  - name: APP_NAME
+    displayName: Name
+    description: A prefix appended to all objects
+    required: true
+  - name: SOURCE_CONTEXT_DIR
+    required: true
+  - name: GIT_URL
+    required: true
+  - name: REPO_REF
+    required: true
+  - name: PROJECT_PREFIX
+    required: true
+  - name: APP_NAME
+    required: true
+  - name: SPRING_PROFILES_ACTIVE
+    required: true
+    value: "cloud,splunk"
+  # TODO value needs to change at some point
+  - name: BAMBORA_PAYMENT_ENDPOINT
+    required: true
+    value: "https://example.com"
+  # TODO value needs to change at some point
+  - name: CRC_ENDPOINT_APPROVED
+    required: true
+    value: "http://approved"
+  # TODO value needs to change at some point
+  - name: CRC_ENDPOINT_DECLINED
+    required: true
+    value: "http://declined"
+  - name: SPLUNK_URL
+    required: true
+    value: "https://dps-splunk:8088"
+  # TODO value needs to change at some point
+  - name: SPLUNK_TOKEN
+    required: true 
+    value: "${SPLUNK_TOKEN}"
```

Comment:
Our unit test don't require any of the custom values to be set, therefor this should be removed

Reply:
I've instructed Adam that since we're going for a 'hello world' to start we're filling all required values with placeholders to begin with.  We'll pull these out once the pipeline is fully operational. Think of it as packing material.

Reply:
removed values